test/Microsoft.ComponentDetection.Detectors.Tests/nuget/NuGetPackagesConfigDetectorTests.cs

```diff
@@ -25,10 +25,29 @@ public async Task Should_WorkAsync()
             .WithFile("packages.config", packagesConfig)
             .ExecuteDetectorAsync();
 
+        scanResult.ResultCode.Should().Be(ProcessingResultCode.Success);
         var detectedComponents = componentRecorder.GetDetectedComponents();
         detectedComponents.Should().NotBeEmpty()
             .And.HaveCount(2)
             .And.ContainEquivalentOf(new DetectedComponent(new NuGetComponent("jQuery", "3.1.1")))
             .And.ContainEquivalentOf(new DetectedComponent(new NuGetComponent("NLog", "4.3.10")));
     }
+
+    [TestMethod]
+    public async Task Should_SkipWithInvalidVersionAsync()
+    {
+        var packagesConfig =
+            @"<?xml version=""1.0"" encoding=""utf-8""?>
+                <packages>
+                    <package id=""jQuery"" version=""3.1.1"" targetFramework=""net46"" />
+                    <package id=""NLog"" version=""
```

Comment:
Ideally we would be able to detect `jQuery`, but NuGet throws an error because the whole file isn't valid XML :(

Reply:
Yeah, and we defer the actual detection to NuGet itself. IMO that's a good thing, because the behaviour is aligned.